kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/client/DefaultFileSystemClientSuite.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.client
+
+import io.delta.kernel.defaults.utils.DefaultKernelTestUtils.getTestResourceFilePath
+import io.delta.kernel.defaults.utils.TestUtils
+import org.apache.hadoop.conf.Configuration
+import org.scalatest.funsuite.AnyFunSuite
+
+import java.io.FileNotFoundException
+import scala.collection.mutable.ArrayBuffer
+
+class DefaultFileSystemClientSuite extends AnyFunSuite with TestUtils {
+
+  val fsClient = new DefaultFileSystemClient(new Configuration())
+
+  test("list from file") {
+    val basePath = fsClient.resolvePath(getTestResourceFilePath("json-files"))
+    val listFrom = fsClient.resolvePath(getTestResourceFilePath("json-files/2.json"))
+
+    val actListOutput = new ArrayBuffer[String]()
+    val files = fsClient.listFrom(listFrom)
+    try {
+      fsClient.listFrom(listFrom).forEach(f => actListOutput += f.getPath)
+    }
+    finally if (files != null) {
+      files.close()
+    }
+
+    val expListOutput = Seq(basePath + "/2.json", basePath + "/3.json")
+
+    assert(expListOutput === actListOutput)
+  }
+
+  test("list from non-existent file") {
+    intercept[FileNotFoundException] {
+      fsClient.listFrom("file:/non-existentfileTable/01.json")
+    }
+  }
+
+  test("resolve path") {
+    val inputPath = getTestResourceFilePath("json-files")
```

Comment:
Could you create an equivalent utility method for `getTestResourceFilePath` in `TestUtils.scala`? that way we can get rid of the `DefaultKernelTestUtils.java` completely at some point.

Reply:
Once this comment is addressed, this PR is good to merge.

Reply:
@vkorukanti I have resolved it, please review it for me